cmd/conveyor/factories.go

```diff
@@ -118,12 +118,12 @@ func newSlackServer(cy *conveyor.Conveyor, c *cli.Context) http.Handler {
 }
 
 func newBuilder(c *cli.Context) builder.Builder {
-	db, err := docker.NewBuilderFromEnv()
+	db, err := codebuild.NewBuilderFromEnv()
```

Comment:
Once this is ready to merge, we'll want to make this configurable with a CLI flag, so you can choose the codebuild backend.